lib/org/openpsa/invoices/handler/invoice/view.php

```diff
@@ -96,7 +96,7 @@ private function populate_toolbar($invoice)
                  && intval($billing_data->sendingoption) == 2) {
                 $buttons[] = $this->build_button('send_by_mail', 'paper-plane');
             }
-            if ($invoice->due > date("Y-m-d") ) {
+            if ((new datetime(date($this->_l10n_midcom->get('short date'), $invoice->deliverydate)))->modify("+14 days") > date("Y-m-d") ) {
```

Comment:
this is unreadable and doesn't make any sense. Pls look how overdue is determined in other parts of the code

Reply:
also, the logic is still wrong: If deliverydate + 14 days > today that means it's not overdue yet